components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java

```diff
@@ -55,6 +56,8 @@
     private static Log log = LogFactory.getLog(SiddhiTryItClient.class);
     private static Gson gson = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
     private SimpleDateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS");
+    Pattern eventPattern = Pattern.compile("(\\S+)=\\[(.*)\\]");
```

Comment:
To further improve this, you can make this a static variable too to further improve memory efficiency. Might not get you any significant results, but still we should be able to adopt this as a best practice.